tests/storm/locustfile.py

```diff
@@ -0,0 +1,67 @@
+import json
+from locust import HttpUser, between, task
+
+from tests.storm.common import execute_async
+
+
+class RookUser(HttpUser):
+    host = "http://localhost:5000"
+    wait_time = between(1, 10)
+
+    @task(4)
+    def capabilities(self):
+        query = "/wps?service=WPS&request=GetCapabilities"
+
+        with self.client.get(
+            query, catch_response=True, name="capabilities"
+        ) as response:
+            if "<ows:Title>Emu</ows:Title>" not in response.text:
+                response.failure("Capabilities does not match expected XML")
+
+    @task(2)
+    def describe_process_subset(self):
+        query = (
+            "/wps?service=WPS&version=1.0.0&request=DescribeProcess&identifier=hello"
+        )
+        with self.client.get(
+            query, catch_response=True, name="describe_process"
+        ) as response:
+            if "<ows:Identifier>hello</ows:Identifier>" not in response.text:
+                response.failure(
+                    "Process description for *hello* does not match expected XML"
+                )
+
+    @task(1)
```

Comment:
The (1) means only 1 hit at a time?

Reply:
the "1" is not necessary ... it is the default ... I will remove it.